### ISSUE TITLE
Problem When pip install tensorflow-compression

### ISSUE THREAD

**Issue by cucrui** [tensorflow/compression#23]
when I install this package by using "pip install tensorflow-compression"
It comes out " Could not find a version that satisfies the requirement tensorflow-compression (from versions: ) No matching distribution found for tensorflow-compression"
How can I fix it. Thanks for reply!

**Comment by jonycgn** [comment]
Hi, which version of Python and Tensorflow are you using, and what platform do you have (Linux/Mac OS/Windows)?

**Comment by cucrui** [comment]
Python3.6 
Tensorflow1.41 
Windows




------------------ 原始邮件 ------------------
发件人: "Johannes Ballé"<notifications@github.com>; 
发送时间: 2019年7月18日(星期四) 晚上9:18
收件人: "tensorflow/compression"<compression@noreply.github.com>; 
抄送: "StevieRay"<1729952693@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [tensorflow/compression] Problem When pip installtensorflow-compression (#23)




Hi, which version of Python and Tensorflow are you using, and what platform do you have (Linux/Mac OS/Windows)?

**Comment by jonycgn** [comment]
Sorry, Windows is currently not supported. Please use one of TensorFlow's Docker images (https://www.tensorflow.org/install/docker) and pip install tensorflow_compression there.

**Comment by bionicles** [comment]
@jonycgn is TF 2.0b1 supported? Also, do you think this project could be used to find the Normalized Compression Distance between arbitrary tensors? I was thinking that might be a nice loss function for our project (molecular dynamics requires a nice way to compare different molecules in different poses) but not sure how to approach it. Sorry to hijack a thread

**Comment by jonycgn** [comment]
No, TF 2.0 is not yet supported. Not too familiar with NCD, but you can use this code to prototype a good general purpose compressor for any data, so presumably you could.

**Comment by bionicles** [comment]
thank you @jonycgn please consider adding TF2.0 and i think if we have NCD we could use it as a new loss function for many many many projects in different areas. A nice extension of tfc potentially
